src/server/simple.cpp

```diff
@@ -22,6 +22,8 @@ DEFINE_string(model_allow_patterns,
               "*.json,*.tiktoken,*.model,*.safetensors",
               "Allow patterns for model files.");
 
+DEFINE_string(token, "", "hugging face token");
```

Comment:
actually, you don't need pass token in this way. a better way is set a environment variable and huggingface_hub will load it automatically. `HUGGING_FACE_HUB_TOKEN`

Reply:
understand！